python-sdk/src/astro/sql/operators/raw_sql.py

```diff
@@ -69,6 +96,52 @@ def make_row_serializable(rows: Any) -> Any:
             return [SdkLegacyRow.from_legacy_row(r) if isinstance(r, SQLAlcRow) else r for r in rows]
         return rows
 
+    @staticmethod
+    def results_as_list(result: ResultProxy) -> list:
+        """
+        Convert the result of a SQL query to a list
+        """
+        return cast(List[Any], result.fetchall())
+
+    @staticmethod
+    def results_as_pandas_dataframe(result: ResultProxy) -> pd.DataFrame:
+        """
+        Convert the result of a SQL query to a pandas dataframe
+        """
+        return pd.DataFrame(result.fetchall(), columns=result.keys())
```

Comment:
we should use Astro's PandasDataframe so it is serializable with Airflow 2.5:

```
from astro.dataframes.pandas import PandasDataframe

return PandasDataframe(result.fetchall(), columns=result.keys())
```

Reply:
Done.